docs/source/user/installation/install-macos.md

```diff
@@ -0,0 +1,82 @@
+# macOS Installation Guide
```

Comment:
uppercase MacOS here, I think...

Reply:
Apple uses lowercase now.  Swanky and hip marketing :)

<img width="1552" alt="vs7g4-image" src="https://user-images.githubusercontent.com/4249591/48099969-6e5fb700-e1ef-11e8-9980-df039cc7ff84.png">

Reply:
heh! Alright then...